### ISSUE TITLE
kitti dataset

### ISSUE THREAD

**Issue by HGao-cv** [jinga-lala/DAMEX#4]
Does the partition of the KITTI dataset used by Damex overlap between the test set and the training set?

**Comment by jinga-lala** [comment]
The partition of KITTI is same as proposed by [UODB benchmark](http://www.svcl.ucsd.edu/projects/universal-detection/)

**Comment by giangdip2410** [comment]
Hi @jinga-lala , the link of [UODB benchmark](http://www.svcl.ucsd.edu/projects/universal-detection/) is dead, could you please share with me the UODB benchmark again?

**Comment by jinga-lala** [comment]
Hi @giangdip2410, unfortunately only the owners of UODB benchmark can release it in this situation. However, you can still install individual datasets from their corresponding locations.
